package.json

```diff
@@ -67,12 +67,12 @@
   },
   "dependencies": {
     "lodash": "^4.17.11",
-    "mobx": "^5.14.2",
```

Comment:
Why not MobX 4?

Reply:
I didn't read the patch notes, just reverted the change.
I think it should work aswell.

Want me to upgrade to 4?

Reply:
No problem, I upgraded to 4 and ran the tests to make sure they pass